src/flux/jinad.ts

```diff
@@ -0,0 +1,335 @@
+import axios, { AxiosInstance } from "axios";
```

Comment:
I think this file should be moved to services directory.

Reply:
Agreed